apps/client/src/common/api/eventsApi.ts

```diff
@@ -50,6 +50,19 @@ export async function requestApplyDelay(eventId: string) {
   return axios.patch(`${rundownURL}/applydelay/${eventId}`);
 }
 
+export type SwapEntry = {
+  from: number;
+  to: number;
+};
```

Comment:
It would, but using `id`s means I have to iterate through the array at least once to get the events I want to swap. I was hoping indexes would save us the relatively expensive operation